### DISCUSSION TITLE
How to generate an Hexagonal stereographic projection ?

### DISCUSSION THREAD

**Discussion by jucego333** [mtex-toolbox/mtex#1127] [Ask Anything]
Dear colleagues 

I would like to generate the standard (0001) stereograph projection like the picture state below. I have been following this thread
https://mtex-toolbox.github.io/SphericalProjections.html 

However, I am not quite sure how to generate images similar to the picture. 

![image](https://user-images.githubusercontent.com/78469467/125283712-3ff4eb00-e34b-11eb-9b50-e469c18df2e1.png)
![image](https://user-images.githubusercontent.com/78469467/125283838-5dc25000-e34b-11eb-884b-5bf9d95f6a21.png)

Best regards

**Comment by kilir** [comment]
Hi,
basically have a look at Miller/plot see here: https://mtex-toolbox.github.io/CrystalDirections.html
e.g.
```
plot(h,'symmetrised','labeled')
```
where `h` is a list of Millers.

Hope that helps.

Cheers,
Rüdiger

**Reply by jucego333** [reply]
Thank you for your reply  @kilir . I read in the book that the center of the stereograph projection can be changed (rotations). In other words,  for the example given in this link https://mtex-toolbox.github.io/CrystalDirections.html  how can we plot the point 
2  1 -3  1 at the middle of the circle and then obtain the standard  2  1 -3  1 stereograph projection?

**Reply by kilir** [reply]
Hi,
this is quite frequently asked. You cannot directly rotate the projection, however you can rotate your directions and just label the points accordingly using `'label','list_of_labels'` instead of `'labeled'`. You have to generate that list of labels and make sure they come in the correct order.
Similar: https://groups.google.com/g/mtexmail/c/G5WSFmIwVd0/m/SpgXz9fhAwAJ

Cheers,
Rüdiger
